### PR TITLE
Add license flag

### PR DESCRIPTION
**Pull request by michaeladda**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "concord",
-  "version": "0.2.1",
+  "version": "0.3.1",
```
Comment (bergundy) [inline package.json]: Better make it a patch version. Doesn't break API.

File: src/types.ts [hunk 2]
```diff
@@ -18,6 +18,7 @@ export interface Package {
   peerDependencies?: {
     [name: string]: string;
   };
+  license?: string;
```
Comment (bergundy) [inline src/types.ts]: I don't think this is optional.

**Review by bergundy — APPROVED** [review]
Thanks!
Please address the 2 minor comments.

File: package.json (hunk 1 above)
Comment (michaeladda) [reply package.json]: this is deliberate. New functionality, even when non breaking, is a minor update, not patch, according to https://semver.org/

File: src/types.ts (hunk 2 above)
Comment (michaeladda) [reply src/types.ts]: changed it to be compulsory and the code to match.

**Review by michaeladda — COMMENTED** [review]
Thanks for the review!

**Pull request merged**